apps/ios/Shared/Model/ChatModel.swift

```diff
@@ -82,19 +86,19 @@ class ItemsModel: ObservableObject {
                 await MainActor.run { showLoadingProgress = true }
             } catch {}
         }
+        let type = ChatType(rawValue: String(chatId.prefix(1)))!
+        let id = Int64(chatId.suffix(from: chatId.index(chatId.startIndex, offsetBy: 1)))!
         Task {
-            if let chat = ChatModel.shared.getChat(chatId) {
-                await MainActor.run { self.isLoading = true }
-//                try? await Task.sleep(nanoseconds: 5000_000000)
-                await loadChat(chat: chat)
-                navigationTimeout.cancel()
-                progressTimeout.cancel()
-                await MainActor.run {
-                    self.isLoading = false
-                    self.showLoadingProgress = false
-                    willNavigate()
-                    ChatModel.shared.chatId = chatId
-                }
+            await MainActor.run { self.isLoading = true }
+            //                try? await Task.sleep(nanoseconds: 5000_000000)
+            await loadChat(type: type, id: id)
+            navigationTimeout.cancel()
+            progressTimeout.cancel()
+            await MainActor.run {
+                self.isLoading = false
+                self.showLoadingProgress = false
+                willNavigate()
+//                ChatModel.shared.chatId = id
```

Comment:
why is this commented?

Reply:
because the id is set in loadChat -> apiLoadMessages

Reply:
aren't there cases when it doesn't change after loading chat?